src/python/pants/backend/scala/test/scalatest_test.py

```diff
@@ -119,22 +126,27 @@ def test_simple_success(rule_runner: RuleRunner, scalatest_lockfile: JVMLockfile
                 class SimpleSpec extends AnyFunSpec {
                   describe("Simple") {
                     it("should be simple") {
-                      assert("Simple".toLowerCase == "simple")
+                      assert("Simple".toLowerCase == "%s")
```

Comment:
couldn't this be an f-string such that in here you would use `{value_in_test}`?

Reply:
I prefer using `%s` in string code blocks so we don't have to double-up the curly-braces. I could change it